### ISSUE TITLE
🐛 More specific error message for top-level await in scripts

### ISSUE THREAD

**Issue by jthegedus** [rome/tools#1015]
## Environment

This is run in an unmodified GitHub Codesapces environment:

```
$ rome rage --summary
Environment Variables
  Object {
    COLORTERM: "truecolor"
    LANG: "C.UTF-8"
    PATH: "/opt/nodejs/12.16.3/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/codespace/workspace/cloudbuild-triggers-sync/node_modules/.bin:/home/code
    space/.dotnet:/home/codespace/.dotnet:/home/codespace/.vscode-remote/bin/5696babbc1dce41788bae04b1a04ad35140bebfb/bin:/home/codespace/.dotnet:/home/codespace/.dotnet:/
    usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin:/home/codespace/.dotnet/tools:/home/c
    odespace/.local/bin:/home/codespace/.npm-global/bin:/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin:/home/codespace/.dotnet/tools:/home/codes
    pace/.local/bin:/home/codespace/.npm-global/bin:/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin:/home/codespace/.dotnet/tools:/home/codespace
    /.local/bin:/home/codespace/.npm-global/bin:/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin:/home/codespace/.dotnet/tools:/home/codespace/.lo
    cal/bin:/home/codespace/.npm-global/bin"
    TERM: "xterm-256color"
    TERM_PROGRAM: "vscode"
  }

User Config
  unset

Rome Version
  10.0.3-beta

Node Version
  12.16.3

Platform
  linux x64 5.3.0-1034-azure

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 24
    columns: 175
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/home/codespace/workspace/cloudbuild-triggers-sync"
    realCwd: "/home/codespace/workspace/cloudbuild-triggers-sync"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 26_046_464
      pid: 1_764
      uptime: 0.327117979
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 26_046_464
        ownedBytes: 0
        ownedFileCount: 0
        pid: 1_764
        uptime: 0.327047775
      }
    ]
  }
```

## Steps To Reproduce

1. create a `.js` script with top-level `await`

The script I used was a modified version of https://github.com/googleapis/nodejs-cloudbuild/blob/master/samples/listBuildTriggers.js where the `async` function contents were lifted to the root of the script:

```shell
✖ Expected a semicolon or a line terminator

    16 │ const cb = new CloudBuildClient();
    17 │ const request = { projectId: TARGET_PROJECT_ID };
  > 18 │ const [result] = await cb.listBuildTriggers(request);
       │                        ^
    19 │ console.info(JSON.stringify(result, null, 2));
```

## Expected Results

The error could be more specific to the usage of `await` instead of missing semicolon.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
